Java/JDI/jdi-cucumber-test-generator/src/main/java/com/epam/test_generator/services/StepSuggestionService.java

```diff
@@ -2,32 +2,138 @@
 
 import com.epam.test_generator.dao.interfaces.StepSuggestionDAO;
 import com.epam.test_generator.dto.StepSuggestionDTO;
-import com.epam.test_generator.dto.DozerMapper;
 import com.epam.test_generator.entities.StepSuggestion;
+import com.epam.test_generator.transformers.StepSuggestionTransformer;
+import com.epam.test_generator.entities.StepType;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
+import javax.annotation.PostConstruct;
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
+import java.util.stream.Collectors;
 
 @Transactional
 @Service
 public class StepSuggestionService {
 
     @Autowired
-    private DozerMapper dozerMapper;
+    private StepSuggestionTransformer stepSuggestionTransformer;
 
     @Autowired
     private StepSuggestionDAO stepSuggestionDAO;
 
+    @PostConstruct
+    private void initializeDB(){
+        List<String> given = Arrays.asList(
+                "I open \"<pageName>\"",
+                "I click on \"<linkName>\" linkI fill \"<fieldName>\" textfield with \"<data>\""
+                );
+
+        List<String> when = Arrays.asList(
```

Comment:
If this data could be changed, It is better to store it in a separate xml or property file